pyproject.toml

```diff
@@ -1,38 +1,39 @@
 [project]
     authors = [
-        { name = "Tomas Sebestik", email = "tomas.sebestik@espressif.com" },
         { name = "Espressif Systems" },
+        { name = "Tomas Sebestik", email = "tomas.sebestik@espressif.com" },
     ]
     classifiers = [
         "Intended Audience :: Developers",
+        "License :: OSI Approved :: Apache Software License",
         "Natural Language :: English",
-        "Operating System :: POSIX",
-        "Operating System :: Microsoft :: Windows",
         "Operating System :: MacOS :: MacOS X",
-        "License :: OSI Approved :: Apache Software License",
+        "Operating System :: Microsoft :: Windows",
+        "Operating System :: POSIX",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
         "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: 3.12",
+        "Programming Language :: Python :: 3.9",
```

Comment:
what's up with this ordering? Is this forced by any of the linters?

Reply:
Yes, Taplo (VS Code plugin "Even Better TOML") sorts items in the TOML list alphabetically - see config file in this project `taplo.toml`. 

This ordering shouldn't matter I think. If this is an issue, we can add a flag to not sort this array, but ...